package.json

```diff
@@ -57,7 +57,7 @@
         "prepare": "husky",
         "prepublish:react": "yarn build:react && sed -i \"\" \"s/react/# react/g\" .gitignore && git commit -am \"Commit React Wrappers\" --no-verify",
         "prestorybook": "wireit",
-        "prestorybook:build": "cem analyze --outdir storybook/",
+        "prestorybook:build": "cem analyze --outdir .storybook/",
```

Comment:
Pre commands should no longer be automatically working in yarn 4 but I think this isn't erroring yet because wireit is holding it together. Not something we need to consider right now but it might worth clarifying and combining required commands so we don't try to run `yarn storybook:build` for example and expect cem to run (but it won't because only `yarn storybook` is a wireit command I believe.

Reply:
same brain wavelengths! also to note we added the `enableScripts: true` to the `yarnrc.yml` to enforce the old behavior of running pre and post :) we added an @todo in a comment above it. this came up as in issue in the 1.1.0 release.